ontology/uco/tool/tool.ttl

```diff
@@ -54,139 +60,54 @@ tool:BuildFacet
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildProject ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildScript ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildVersion ;
-		]
-		;
+	rdfs:seeAlso observable:SoftwareBuildFacet ;
+	sh:message "uco-tool:BuildInformationType is deprecated.  Please migrate usage of this class to uco-observable:SoftwareBuildFacet."@en ;
+	sh:not [
+		a sh:NodeShape ;
+		sh:class tool:BuildInformationType ;
+	] ;
 	sh:targetClass tool:BuildInformationType ;
 	.
 
 tool:BuildUtilityType
 	a
 		owl:Class ,
+		owl:DeprecatedClass ,
 		sh:NodeShape
 		;
 	rdfs:subClassOf core:UcoInherentCharacterizationThing ;
 	rdfs:label "BuildUtilityType"@en ;
 	rdfs:comment "A build utility type characterizes the tool used to convert from source code to executable code for a particular version of software."@en ;
-	sh:property
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:buildUtilityName ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:cpeid ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:swid ;
-		]
-		;
+	rdfs:seeAlso observable:BuildUtilityFacet ;
+	sh:message "uco-tool:BuildUtilityType is deprecated.  Please migrate usage of this class to uco-observable:BuildUtilityFacet."@en ;
+	sh:not [
+		a sh:NodeShape ;
+		sh:class tool:BuildUtilityType ;
+	] ;
 	sh:targetClass tool:BuildUtilityType ;
 	.
 
 tool:CompilerType
 	a
 		owl:Class ,
+		owl:DeprecatedClass ,
 		sh:NodeShape
 		;
 	rdfs:subClassOf core:UcoInherentCharacterizationThing ;
 	rdfs:label "CompilerType"@en ;
 	rdfs:comment "A compiler type is a grouping of characteristics unique to a specific program that translates computer code written in one programming language (the source language) into another language (the target language). Typically a program that translates source code from a high-level programming language to a lower-level language (e.g., assembly language, object code, or machine code) to create an executable program. [based on https://en.wikipedia.org/wiki/Compiler]"@en ;
-	sh:property
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:compilerInformalDescription ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:cpeid ;
-		] ,
-		[
-			sh:datatype xsd:string ;
-			sh:maxCount "1"^^xsd:integer ;
-			sh:nodeKind sh:Literal ;
-			sh:path tool:swid ;
-		]
-		;
+	rdfs:seeAlso observable:CompilerFacet ;
+	sh:message "uco-tool:CompilerType is deprecated.  Please migrate usage of this class to uco-observable:CompilerFacet."@en ;
```

Comment:
`CompilerFacet` was not described directly in the proposal, but was referenced in some of the migration notes.

@sbarnum - Do you have an `rdfs:comment` for this class?